common/locales/de.json

```
@@ -162,8 +162,10 @@
     "materialItem": {
       "fields": {
         "article": "Artikel",
+        "list": "Liste",
         "quantity": "Menge",
-        "unit": "Einheit"
+        "reference": "Aktivität / Abschnitt",
+        "unit": "Mass"
```

```suggestion
        "unit": "Masseinheit"
```
Maybe @pmattmann knows better how to indicate kg/stück/liter/Löffel/Säcke?

---

I chose this as it is short 🤷🏼‍♂️
But the whole thing needs to be reworked anyway. So it could be that we eliminate the unit label anyway